src/SolutionTemplate/UnoLibraryTemplate/CrossTargetedLibrary.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/1.6.46">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;xamarinios10;monoandroid80;uap10.0</TargetFrameworks>
+		
+		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
```

Comment:
define __ iOS __ and __ ANDROID __ in the same fashion?

Reply:
No, those are already defined by Xamarin. __WASM__ is special because it's not supported by default (it's not a known VS target)